### PR TITLE
Added set up files

### PR DESCRIPTION
**Pull request by acookson91**

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'sinatra/base'
+
+class makers_bnb < Sinatra::Base
+  get '/' do
```
Comment (paulalexrees) [inline app/app.rb]: Get rid of this rubbish

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ENV['RACK_ENV'] = 'test'
+
+require File.join(File.dirname(__FILE__), '..', 'app/app.rb')
+
+require 'capybara'
+require 'capybara/rspec'
+require 'rspec'
+
+Capybara.app = makers_bnb
+# hmm - should be generated by `rspec --init` directly rather than copy paste ...
```
Comment (paulalexrees) [inline spec/spec_helper.rb]: These comments are litter

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# A sample Gemfile
```
Comment (hanfak) [inline Gemfile]: get rid

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
```
Comment (hanfak) [inline spec/spec_helper.rb:12]: Sort all these comments out

File: spec/spec_helper.rb [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  #   - http://teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/
+  #   - http://myronmars.to/n/dev-blog/2014/05/notable-changes-in-rspec-3#new__config_option_to_disable_rspeccore_monkey_patching
+  config.disable_monkey_patching!
+
+  # This setting enables warnings. It's recommended, but in some cases may
+  # be too noisy due to issues in dependencies.
+  config.warnings = true
+
+  # Many RSpec users commonly either run the entire suite or an individual
+  # file, and it's useful to allow more verbose output when running an
```
Comment (hanfak) [inline spec/spec_helper.rb]: not impressed

File: spec/spec_helper.rb (hunk 2 above)
Comment (claudiagreen) [inline spec/spec_helper.rb]: What tf is all this commented code?

File: app/app.rb [hunk 6]
```diff
@@ -0,0 +1,10 @@
+require 'sinatra/base'
+
+class makers_bnb < Sinatra::Base
```
Comment (paulalexrees) [inline app/app.rb]: Class name should be MakersBnb

**Comment by paulalexrees** [comment]
:shipit:

**Comment by claudiagreen** [comment]
:shipit:

**Comment by acookson91** [comment]
closes #1 
closes #13

**Pull request merged**